### ISSUE TITLE
Question: Why is this repo not a fork as per GitHub? 

### ISSUE THREAD

**Issue by Semnodime** [ferdium/ferdium-app#1692]
### Your issue

In the README.md it is mentioned that ferdium is a "Hard-fork of Franz".

I wonder, for which reason it has not been published to formally be recognized as fork as well, as in a [GitHub fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo).

**Comment by SpecialAro** [comment]
Please read this: https://github.com/ferdium

**Comment by Semnodime** [comment]
> @vraravam Please read this: https://github.com/ferdium

I read the README.md, fully. It does not answer the question. That's why I opened the issue to begin with.

**Comment by SpecialAro** [comment]
Ferdi forked Franz, Ferdium forked Ferdi.

Although, we were forced to do a hard fork, given that at the time we started Ferdi code was taken down by the sole code owner (which was the trigger for contributors to push their local forks to a new project).

That is why I think GitHub doesn't state this repo as a fork.

**Comment by Semnodime** [comment]
I just tested the behavior of GitHub and noticed that if a public repository is forked and then deleted, the fork does not reference the original repository anymore. So (at least visually and AFAICT) it wouldn't matter if the fork happened formally or not.

**Comment by SpecialAro** [comment]
So, I think you just prove what I was saying right?

**Comment by Semnodime** [comment]
I wasn't sure whether a new repository was created based on the code of a fork / local clone or whether a fork on GitHub has had its ownership transferred to ferdium.

> @SpecialAro I think you just prove what I was saying

After reading your answer I was still not sure, that's why I experimented with GitHub myself.

I suspected a formal fork to be visually different, displaying an indicator that the repository is a fork (even if the original repository has been deleted since). AFAICT, this is not the case though, so whether a repository is newly created or a fork of a since then deleted repository does not make a difference, that is visually, at least.
